src/frontend/apps/desk/next.config.js

```diff
@@ -1,6 +1,33 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   output: 'export',
+  webpack(config) {
```

Comment:
it looks gooooood, I will be more than happy discussing it orally for my personal culture.